pytential/linalg/skeletonization.py

```diff
@@ -52,6 +52,50 @@
 
 # {{{ wrangler
 
+# Adding weights to proxy evaluation: Why? How? Huh?
+# --------------------------------------------------
+#
+# We have a couple of interaction evaluations that need to happen for the
+# proxy-based skeletonization, namely
+# 1. with the proxies + neighbors when the current cluster is the source
+# 2. with the proxies + neighbors when the current cluster is the target
+#
+# The operator that we want to skeletonize at the end of the day is the full
+# layer potential, that contains quadrature weights and area elements. That
+# means that it is in our interest to include some of that information in
+# the proxy + neighbor interaction matrices that get passed to the ID
+# (interpolative decomposition), so that it has a better chance at guessing a
+# good range!
```

Comment:
Maybe rephrase this to emphasize weighting of relative error?

Reply:
Not quite sure what you had in mind here?

Reply:
I feel that the intutition we had built was that the ID will ultimately seek to squeeze the matrix norm of the error below the supplied bound, i.e. a relative error measure. That's easier for columns with small norms than those with big norms, and so it'll "make more effort" to make big-norm columns "happy" than others. But for our application, the unequal weighting is spurious, that's why we try to fix it.

I was hoping the comment narrative could reflect that train of thought.

Reply:
Ah, right! That totally slipped my mind. I added something like that in 
https://github.com/inducer/pytential/compare/0617bdafd0a7a37c2e162b752371ab908e456ad5..2e897125c3e199988600d73c65ca28c1a7d55c1c
(let me know if it's not clear enough!)

Reply:
That works, thanks!